README.md

```diff
@@ -1034,6 +1034,14 @@ All statements above will return false if used with `===`
 
 <br/><br/><br/>
 
+## ![✔] 7.2. Replicate the Node process
+
+ **TL;DR:** If you really need to use some heavy computational work in your application and you can’t make use of the [non-blocking event loop](https://nodejs.org/en/docs/guides/dont-block-the-event-loop), and if you have a managed server with more than a single CPU, you should consider using the native Node.js `cluster` module or a library like [PM2](http://pm2.keymetrics.io/docs/usage/cluster-mode). Cluster mode will allow you to take advantage of your CPU resources to replicate the single thread when the Event Loop is blocked.
```

Comment:
Replication gains better performance even for IO-only work - I tried once to run 1 vs 4 express apps that just serve requests (no CPU work) and saw 20% improvements for 4 processes

I would generalize the advice - if you have more than 1 vCPU, replicate a Node.js process per each